modules/gdscript/gdscript_parser.cpp

```diff
@@ -2975,6 +2975,13 @@ GDScriptParser::ExpressionNode *GDScriptParser::parse_dictionary(ExpressionNode
 					case GDScriptTokenizer::Token::EQUAL:
 						dictionary->style = DictionaryNode::LUA_TABLE;
 						break;
+					case GDScriptTokenizer::Token::COMMA:
```

Comment:
I don't think that `parse_dictionary` should call to `parse_set`. Better use another method `parse_dictioanry_or_set` that delegates to both of the methods.